bundle/manifests/curator-operator.clusterserviceversion.yaml

```diff
@@ -300,10 +327,17 @@ spec:
                     memory: 64Mi
                 securityContext:
                   allowPrivilegeEscalation: false
+                volumeMounts:
+                - mountPath: /etc/secret-volume
+                  name: db-secret
               securityContext:
                 runAsNonRoot: true
               serviceAccountName: curator-operator-controller-manager
               terminationGracePeriodSeconds: 10
+              volumes:
+              - name: db-secret
+                secret:
+                  secretName: db-secret-file
```

Comment:
(not blocking/follow-up): Let's make the secret mount read-only or restrict the permissions of the mount. See https://kubernetes.io/docs/concepts/storage/volumes/#secret for more information.